src/module.c

```diff
@@ -119,6 +119,8 @@ int TSDB_info(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
         RedisModuleDictIter *iter = RedisModule_DictIteratorStartC(series->chunks, ">", "", 0);
         Chunk_t *chunk = NULL;
         int chunkCount = 0;
+        RedisModule_ReplyWithSimpleString(ctx, "keySelfName");
```

Comment:
@OfirMos I think its better to move this into the debug section, this is an internal implementation detail that we want to expose to be able to debug the issue. we don't want this to be part of our API.

Reply:
@danni-m It's already in the debug section